src/components/admin/app-breadcrumbs/AppBreadcrumbs.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="row">
+    <div class="col-md-12">
+      <vuestic-breadcrumbs :breadcrumbs="normalizedBreadcrumbs"/>
```

Comment:
:breadcrumbs="breadcrumbsConfig" (or you can find a better name :)), :current-path=""